pkg/cake/cake.go

```diff
@@ -0,0 +1,63 @@
+package cake
+
+type Cake struct {
+	Flour     int // grams of flour
+	Eggs      int // number of eggs
+	Butter    int // grams of butter
+	Chocolate int // grams of chocolate
+	Sugar     int // grams of sugar
+	Carrots   int // number of carrots
+	Tomatoes  int // number of tomatoes
```

Comment:
For the variables in grams: wouldn't it make more sense to be a float? e.g Butter may be 1.2 grams or so.